Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'solidus', github: 'solidusio/solidus', branch: 'v1.1'
```

Comment:
The `Gemfile` is for the local development at the gem itself and is not related to the `.gemspec` that's read when using the gem in your app.